File: src/get-greeting.ts
import { DateTime } from "luxon";

export default function getGreeting(): [string, string] {
  const { hour } = DateTime.local({ zone: "Europe/Amsterdam" });

  return hour < 18
    ? ["Goedemiddag! ☀️", "Fijne dag verder!"]
    : ["Goedenavond! 🌙", "Geniet van je avond!"];
}

File: internal/get_greeting.go
package internal

import "github.com/heyajulia/energieprijzen/internal/datetime"

func GetGreeting() (hello, goodbye string) {
	hour := datetime.Amsterdam().Hour()

	if hour < 18 {
		hello = "Goedemiddag! ☀️"
		goodbye = "Fijne dag verder!"
	} else {
		hello = "Goedenavond! 🌙"
		goodbye = "Geniet van je avond!"
	}

	return
}
